contrib/py/keygen/keygen.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+"""
+keygen tool for lokinet
+"""
+
+from argparse import ArgumentParser as AP
+from base64 import b32encode
+
+import pysodium
```

Comment:
I think that the pynacl module would be a better choice here (despite the name, it is a libsodium wrapper) as it is far more commonly used than pysodium.

Reply:
it's easier with pysodium. pynacl's interface is too high level for what i need, it gets waaaay to verbose.